features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/callback/RecorderWindowCallback.kt

```diff
@@ -54,16 +56,24 @@ internal class RecorderWindowCallback(
                 copy.recycle()
             }
         } else {
-            // TODO: RUMM-2397 Add the proper logs here once the sdkLogger will be added
-            // sdkLogger.errorWithTelemetry("Received MotionEvent=null")
+            internalLogger.log(
+                InternalLogger.Level.ERROR,
+                InternalLogger.Target.TELEMETRY,
+                { MOTION_EVENT_WAS_NULL_ERROR_MESSAGE },
+                null
+            )
         }
 
         @Suppress("SwallowedException")
         return try {
             wrappedCallback.dispatchTouchEvent(event)
         } catch (e: Throwable) {
-            // TODO: RUMM-2397 Add the proper logs here once the sdkLogger will be added
-            // sdkLogger.errorWithTelemetry("Wrapped callback failed processing MotionEvent", e)
+            internalLogger.log(
+                InternalLogger.Level.ERROR,
+                InternalLogger.Target.TELEMETRY,
```

Comment:
Same thing the wrapped callback is customer code, so there's nothing we can do if we detect some errors. In this case, probably a `USER` message would be best.